docs/superpowers/plans/2026-08-28-walletconnect-joey-signin.md

```
@@ -0,0 +1,1175 @@
+
+def test_config_reports_block_when_enabled(monkeypatch):
+    monkeypatch.setattr(config, "REOWN_PROJECT_ID", "pid123")
+    monkeypatch.setattr(config, "WC_CHAIN", "xrpl:1")
+    monkeypatch.setattr(config, "WC_SURFACES", frozenset({"web"}))
+    body = json.loads(_run(app.handle_config(None)).text)
+    assert body["walletconnect"] == {"project_id": "pid123", "chain": "xrpl:1", "surfaces": ["web"]}
+```
+
+- [ ] **Step 2: Run** → AttributeError.
+- [ ] **Step 3: Implement** per interfaces; `handle_config` ignores its `request` today (it does — the test passes `None`; if it reads request, use the `_Req` fake from Task 6). Add the two env lines to `CLAUDE.md`'s env block:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)